File: trunk/server/net/Protocols/ActionRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 卡牌行动 协议:23
    /// </summary>
	[Proto(value=23,description="卡牌行动")]
	[ProtoContract]
	public class ActionRequest
	{
        /// <summary>
        ///  房间id
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;
        /// <summary>
        ///  出牌id
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string uniqueId;
        /// <summary>
        ///  目标
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string targetId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/ActionRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = 23, description = "卡牌行动")
public class ActionRequest
{
	@Protobuf(description = "房间id")
	private String roomId;
	@Protobuf(description = "出牌id")
	private String uniqueId;
	@Protobuf(description = "目标")
	private String targetId;
}
